semantics_analysis/relation_extractor/ontology_utils.py

```diff
@@ -1045,9 +1045,9 @@
                 'description': 'модель не используется для решения указанной задачи',
                 'examples': [
                     {
-                        'text': 'Стало очевидно, что обычный многослойный перцептрон не способен обрабатывать последовательности элементов',
-                        'Model': 'многослойный перцептрон',
-                        'Task': 'обрабатывать последовательности элементов'
+                        'text': 'NlpCraft IDL - это декларативный язык, позволяющий создавать определения интентов для их последующего использования в моделях Apache NlpCraft.',
+                        'Model': 'Apache NlpCraft',
+                        'Task': 'определения интентов'

```

Comment:
The `Apache NLPCraft` is used for the intent recognition task: https://nlpcraft.apache.org/intent-matching.html.

They build syntaxis patterns using `NlpCraft IDL` for this.